src/AlertsManagement/AlertsManagement/help/Set-AzAlertProcessingRule.md

```diff
@@ -435,7 +435,7 @@ Accept wildcard characters: False
 
 ### -ScheduleReccurence2EndTime
 Reccurence Start Time in the timezone of ScheduleTimeZone parameter. Format 06:00:00
-Should be mentioned in case of Reccurent  Schedule - Daily, Weekly or Monthly.
+Should be mentioned in case of Recurrent  Schedule - Daily, Weekly or Monthly.
```

Comment:
One more space between Recurrent and Schedule. Could you please remove it?

Reply:
@vrindagoel-microsoft  the spaces were not part of my change and there are more grammar / markdown changes to be done but i tried to limit it to spelling. There are over 10k potential grammar / markdown / spelling as this is a huge repo. Do you prefer i have all changes in a page at one go ?

Reply:
As example, see the double spaces on lines 141, 156, 173, 190, 276, 293, 310, 327, 406, 486, 581, and even those are not consistent between occurrences.